gpu-lab.md

```diff
@@ -7,21 +7,41 @@ title: Welcome to the CHTC GPU Lab
 The CHTC GPU Lab is a [UW2020-funded project][uw2020] that will expand shared GPU 
 computing infrastructure at UW-Madison.  It will include:
 
-* a pool of shared GPU servers managed by CHTC 
-* a community of technical experts
-* a library of sharable software and documentation
+<table>
```

Comment:
Are you using HTML instead of Markdown for tables in order to control column widths more precisely and center one column?

Reply:
Yes. Even this is a hack - I'm not very good at managing image size/alignment on the web in general. 

The table isn't necessary - I think this would work just fine as the original list.

Reply:
If we can get the formatting to work, the table with graphics is nice.